.vscode/launch.json

```diff
@@ -1,31 +1,28 @@
 {
-  // Use IntelliSense to learn about possible attributes.
-  // Hover to view descriptions of existing attributes.
-  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
   "version": "0.2.0",
   "configurations": [
     {
-      "name": "Home Assistant",
+      "name": "Debug Home Assistant",
       "type": "python",
       "request": "launch",
       "module": "homeassistant",
       "justMyCode": false,
       "args": ["--debug", "-c", "config"],
-      "preLaunchTask": "Compile English translations"
+      "preLaunchTask": "Compile English translations",
+      "description": "Debug Home Assistant locally with English translations compilation."
```

Comment:
It is running, debugging is optional.

The English translations details isn't needed or helpful here.